### ISSUE TITLE
RCNN __init__() error

### ISSUE THREAD

**Issue by edisel** [broadinstitute/keras-rcnn#177]
Hi everyone,
I have an error when running the following code :
```
import keras
from keras.layers import Input
from keras_rcnn.models import RCNN
img_in = Input((None, None, 3))
rcnn_model = RCNN(img_in, 2)
```

The error message is : 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() missing 1 required positional argument: 'regions_of_interest'

I tested on two different machines : 
* Machine 1 : windows. Python v 3.5.1 ; Keras 2.1.3 ;  keras-rcnn : 0.0.2
* Machine 2 : CentOS 7. Python v 3.4  ;  Keras 2.1.3 ; keras-rcnn : 0.0.2

I have installed keras-rcnn using the command pip install.
How can I fix that ? Thank you !

**Comment by jhung0** [comment]
Did you try downloading from source? What does line 1 refer to?

**Comment by edisel** [comment]
1) Yes, if I download the .zip from source and install it on CentOS 7, using the command : pip3 install keras-rcnn-master.zip , then I can't import keras-rcnn module in python. I have the following error : 
```
>>> from keras_rcnn.models import RCNN
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/chem/selwa/venvs/tensorflow/lib/python3.4/site-packages/keras_rcnn/models/__init__.py", line 1, in <module>
    from ._rcnn import RCNN
  File "/home/chem/selwa/venvs/tensorflow/lib/python3.4/site-packages/keras_rcnn/models/_rcnn.py", line 10, in <module>
    import keras_rcnn.layers
  File "/home/chem/selwa/venvs/tensorflow/lib/python3.4/site-packages/keras_rcnn/layers/__init__.py", line 1, in <module>
    from .losses import (RCNN, RCNNMaskLoss, RPN)
  File "/home/chem/selwa/venvs/tensorflow/lib/python3.4/site-packages/keras_rcnn/layers/losses/__init__.py", line 56
    return {**super(RCNN, self).get_config(), **configuration}
             ^
SyntaxError: invalid syntax
```
I don't have that error if I install it using the command : pip3 install keras-rcnn

2) I guess line 1 refers to the first line of the module RCNN...

**Comment by maryamag85** [comment]
same issue here in ubuntu 16.04 LST python3

**Comment by edisel** [comment]
I found something similar : https://github.com/broadinstitute/keras-rcnn/issues/140
Apparently it needs at least python 3.6 to run.

**Comment by jmd9019** [comment]
@edisel was the issue solved because I am getting same error 
I am running in jupyter and my python version is 3.6

**Comment by edisel** [comment]
Yes, installing python3.6 solved the problem for me. I also installed keras-rcnn from git repository because the install using the wheel led to "init()" error.

**Comment by jmd9019** [comment]
Still facing the same issue, these are steps how I installed please correct me if have missed something or installed in wrong way
I am running in jupyter with python 3.6 anaconda environment in aws p2 instance.

First I updated keras and tensor, then downloaded zip file and installed using pip installer
But still facing the issue.

**Comment by edisel** [comment]
I'm using neither jupyter, nor anaconda, so I can't help you with this. I installed python36u and created virtualenv as described there : 
  https://janikarhunen.fi/how-to-install-python-3-6-1-on-centos-7.html
Then I ran virtualenv, and installed tensorflow, keras, etc, with pip command.
Finally I installed keras-rcnn from git repository : 
pip install git+git://github.com/broadinstitute/keras-rcnn.git 
or by downloading the .zip file and installing it with pip command (I don't remember how I proceeded this time).
Have you tried to remove keras-rcnn and keras and reinstall them ?
